src/lib/openapi/spec/api-token-schema.ts

```diff
@@ -6,47 +6,72 @@ export const apiTokenSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['username', 'type'],
+    description: 'An overview of an API token.',
     properties: {
         secret: {
             type: 'string',
+            description: 'The token used for authentication.',
+            example: 'project:environment.xyzrandomstring',
         },
         username: {
             type: 'string',
+            description: 'The name of the token',
+            example: 'some-user',
         },
         type: {
             type: 'string',
             enum: Object.values(ApiTokenType),
+            description: 'The type of API token',
+            example: 'client',
         },
         environment: {
             type: 'string',
+            description: 'The environment the token has access to.',
+            nullable: true,
+            example: 'development',
         },
         project: {
             type: 'string',
+            description: 'The project this token belongs to.',
+            example: 'developerexperience',
         },
         projects: {
             type: 'array',
+            description:
+                'The list of projects this token has access to. If the token has access to specific projects they will be listed here. If the token has access to all projects it will be represented as [`*`]',
```

Comment:
Just for my understanding: can this be an empty list (if the token has access to no projects)?

Reply:
No - for project api-tokens it has to be exactly 1 - the specific project it is tied to.
For other api-tokens it gets the ALL parameter if project/list of projects isn't provided.

This should perhaps be reflected in the documentation.

Reply:
Right. Could you put that in the description? That is the bit of docs that is the closest to the actual API and is at least where I would look.